libs/Annotator.py

```diff
@@ -28,16 +32,21 @@ async def annotate(self, spectra, jobs, repeat=False):
                 if job.target not in metadata:
                     try:
                         metadata, cache = await self.execute_job_with_cache(job, metadata, cache)
+                        logger.add_success()
                         if repeat:
                             added_metadata = True
-                    except DataNotAvailable:
-                        pass  # TODO log data for conversing missing in given metadata
-                    except ConversionNotSupported:
-                        pass  # TODO log this type of conversion is not supported by the service or service unknown
-                    except DataNotRetrieved:
-                        pass  # TODO log no data were retrieved
+                    except (SourceAttributeNotAvailable, ConversionNotSupported,
+                            TargetAttributeNotRetrieved, UnknownResponse) as exc:
+                        warning.add_warning(exc)
+                    except ServiceNotAvailable:
+                        warning.add_warning(ServiceNotAvailable(f'Service {job.service} not available.'))
                 else:
-                    pass  # TODO: log - data already present
+                    warning.add_warning(Exception(f'Conversion ({job.service}) {job.source} -> {job.target}: Requested '
```

Comment:
Maybe drop level to `debug` or `info`?

Reply:
Changed to `info`; added option to ignore `info` level in 118cfa5.